tests/get-routes.test.ts

```diff
@@ -1,11 +1,9 @@
 import path from "node:path";
 import url from "node:url";
-import { test } from "uvu";
-import * as assert from "uvu/assert";
 
 import routesArrayFixture from "./fixtures/routes-array.js";
 
-import { getRoutes } from "../dist/node";
+import { getRoutes } from "../lib/node";
```

Comment:
I was having some serious trouble getting the tests to run on the output code. This is true of both on main and this branch. I know ideally you'd like to test against the actual generated code, but perhaps for unit tests this isn't really necessary. Instead of that I think it'd be good to add some E2E tests to the example to ensure the package format is correct and everything works in a browser. I didn't do that here, but that's what I think we should do.